src/stores/data.js

```diff
@@ -3,47 +3,170 @@ import { getNetworkDataById, userAddress } from "./web3";
+    { key: "last3roundavgdcv", value:"Rnd3 DCV", display: (volume) => '$' + volume, tooltip: descriptions.default.tooltip_datafarming_3_round_consume },
+    {
+      key: "roundvolume",
+      value: "Rnd Volume",
+      display: (volume) => '$' + volume,
+      tooltip: descriptions.default.tooltip_datafarming_round_consume
+    },
+    { key: "lastroundvolume", value: "Last Rnd Volume", display: (volume) => '$' + volume, tooltip: descriptions.default.tooltip_datafarming_last_round_consume },
+    { key: "myallocation", value:"My Allocation", tooltip: descriptions.default.tooltip_datafarming_my_allocation },
+  ],
+  'apy': [
+    { key: "network", value: "Network" },
+    { key: "title", value: "Title", tooltip: descriptions.default.tooltip_datafarming_title},
+    { key: "symbol", value: "Symbol" },
+    { key: "last10roundavgapy", value: "Rnd10 APY", display: (apy) => parseFloat(apy ? apy * 100 : 0).toFixed(2) + '%', tooltip: descriptions.default.tooltip_datafarming_10_round_asset_APY },
+    { key: "last5roundavgapy", value: "Rnd5 APY", display: (apy) => parseFloat(apy ? apy * 100 : 0).toFixed(2) + '%', tooltip: descriptions.default.tooltip_datafarming_5_round_asset_APY },
+    { key: "last3roundavgapy", value: "Rnd3 APY", display: (apy) => parseFloat(apy ? apy * 100 : 0).toFixed(2) + '%', tooltip: descriptions.default.tooltip_datafarming_3_round_asset_APY },
+    { key: "roundapr", value: "Rnd APR", display: (roundAPR) => parseFloat(roundAPR ? roundAPR * 100 : 0).toFixed(2) + '%', tooltip: descriptions.default.tooltip_datafarming_current_round_asset_APR },
+    { key: "roundapy", value: "Rnd APY", display: (apy) => parseFloat(apy ? apy * 100 : 0).toFixed(2) + '%', tooltip: descriptions.default.tooltip_datafarming_current_round_asset_APY },
+    { key: "lastroundapy", value: "Last Rnd APY", display: (apy) => parseFloat(apy ? apy * 100 : 0).toFixed(2) + '%', tooltip: descriptions.default.tooltip_datafarming_last_round_asset_APY},
+    { key: "roundyield", value: "Rnd Yield", display: (roundYield) => parseFloat(roundYield ? roundYield * 100 : 0).toFixed(2) + '%', tooltip: descriptions.default.tooltip_datafarming_current_round_asset_yield },
+    { key: "myallocation", value:"My Allocation", tooltip: descriptions.default.tooltip_datafarming_my_allocation },
+  ]
+}
+
+export const defaultColumns = {
+  'alloc': ["Title", "Rnd5 Avg Alloc", "Rnd3 Avg Alloc", "Rnd Alloc", "Curr Alloc", "My Allocation"],
+  'dcv': ["Title", "Rnd10 DCV", "Rnd5 DCV", "Rnd3 DCV", "Last Rnd Volume", "Rnd Volume", "My Allocation"],
+  'apy': ["Title", "Rnd5 APY", "Rnd3 APY", "Last Rnd APY", "Rnd APY", "Rnd Yield", "My Allocation"],
+}
 
-export const defaultColumns = ["Title", "RoundVolume", "RoundAPY", "LastRoundAPY", "CurrentAllocation", "MyAllocation", "OwnerAllocation", "MyVeOcean"]
+async function getDatasetsAvgs(api,roundNumber) {
```

Comment:
Looks like there are a few issues with this request:
- the returned data is too big and can't be received
- the average doesn't seem to be right